monkey/monkey_island/cc/resources/local_run.py

```diff
@@ -20,7 +20,7 @@
 logger = logging.getLogger(__name__)
 
 
-def run_local_monkey():
+def run_local_monkey(dest_dir):
```

Comment:
This whole method should end up in a service. Not sure if this is "in scope" of this PR though